homeassistant/components/snips.py

```diff
@@ -0,0 +1,120 @@
+"""
+Support for Snips on-device ASR and NLU.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/snips/
+"""
+from homeassistant.helpers import template, script, config_validation as cv
+import homeassistant.loader as loader
+import voluptuous as vol
+import copy
+import json
+import logging
+
+DOMAIN = 'snips'
+DEPENDENCIES = ['mqtt']
+CONF_TOPIC = 'topic'
+DEFAULT_TOPIC = '#'
```

Comment:
Since message_received only cares about messages from `hermes/nlu/intentParsed`, better to just listen to that topic instead and not allow it to be configured?